### PR TITLE
Change axis helper to make it support not approximate values for X axis

### PR DESCRIPTION
**Pull request by vtkalek**

### CODE REVIEW

File: src/axis/axis.ts [hunk 1]
```diff
@@ -781,6 +783,8 @@ module powerbi.extensibility.utils.chart.axis {
 
         let scalarDomain = dataDomain ? dataDomain.slice() : null;
         let bestTickCount = maxTicks;
+        if (disableNice)
```
Comment (ignatvilesov) [inline src/axis/axis.ts]: Could you please add the curly braces around this condition.

File: src/axis/axis.ts [hunk 2]
```diff
@@ -801,6 +805,7 @@ module powerbi.extensibility.utils.chart.axis {
         }
         else {
             if (isScalar && dataDomain.length > 0) {
+                if (!disableNice)
```
Comment (ignatvilesov) [inline src/axis/axis.ts]: Could you please add the curly braces around this condition.

File: lib/index.d.ts [hunk 3]
```diff
@@ -221,6 +221,8 @@ declare module powerbi.extensibility.utils.chart.axis {
         is100Pct?: boolean;
```
Comment (ignatvilesov) [inline lib/index.d.ts]: Could you please increment version of the package.json.

File: lib/index.js [hunk 4]
```diff
@@ -789,14 +789,16 @@ var powerbi;
...
+                        var pixelSpan = options.pixelSpan, dataDomain = options.dataDomain, metaDataColumn = options.metaDataColumn, outerPadding = options.outerPadding || 0, isScalar = !!options.isScalar, isVertical = !!options.isVertical, forcedTickCount = options.forcedTickCount, categoryThickness = options.categoryThickness, shouldClamp = !!options.shouldClamp, maxTickCount = options.maxTickCount, disableNice = options.disableNice;
                         var dataType = getCategoryValueType(metaDataColumn, isScalar);
                         var maxTicks = isVertical ? getRecommendedNumberOfTicksForYAxis(pixelSpan) : getRecommendedNumberOfTicksForXAxis(pixelSpan);
                         if (maxTickCount &&
                             maxTicks > maxTickCount)
                             maxTicks = maxTickCount;
                         var scalarDomain = dataDomain ? dataDomain.slice() : null;
                         var bestTickCount = maxTicks;
+                        if (disableNice)
```
Comment (AviSander) [inline lib/index.js]: This changes the logic for the cases below. Have you verified each does not break?

Comment (vtkalek) [inline lib/index.js]: Yes i checked in different variants like disableNice is undefined, disableNice = true and disableNice = false - all is working.

**Pull request merged**